task_sender/pkg/da_utils.go

```diff
@@ -0,0 +1,13 @@
+package pkg
+
+func SplitIntoChunks(proof []byte, chunkSize uint64) [][]byte {
+	chunks := make([][]byte, 0)
+	for i := 0; i < len(proof); i += int(chunkSize) {
+		end := i + int(chunkSize)
+		if end > len(proof) {
+			end = len(proof)
+		}
+		chunks = append(chunks, proof[i:end])
+	}
+	return chunks
```

Comment:
small nit, it may be better to calculate the length of the proof just once, store it in a variable and use that instead

Reply:
Changed on 8b1b4139534cff08ee77999421a79b87ff98999c